cmd/login/cmd.go

```diff
@@ -121,6 +122,13 @@ func init() {
 		"Enables insecure communication with the server. This disables verification of TLS "+
 			"certificates and host names.",
 	)
+	flags.StringVar(
+		&args.rhRegion,
+		"rh-region",
+		"",
+		"OCM region identifier.",
```

Comment:
Is `OCM` a known acronym to our customers or is it better to use the full `Openshift Cluster Manager` description?